Classes/Networkteam/SentryClient/ErrorHandler.php

```diff
@@ -100,11 +105,9 @@ protected function setUserContext() {
 			if ($account !== NULL) {
 				$userContext['username'] = $account->getAccountIdentifier();
 			}
-			$party = $securityContext->getParty();
-			if ($party instanceof Person && $party->getPrimaryElectronicAddress() !== NULL) {
-				$userContext['email'] = (string)$party->getPrimaryElectronicAddress();
-			} elseif ($party !== NULL && ObjectAccess::isPropertyGettable($party, 'emailAddress')) {
-				$userContext['email'] = (string)ObjectAccess::getProperty($party, 'emailAddress');
+			$externalUserContextData = $this->userContextService->getUserContext($securityContext);
+			if ($externalUserContextData !== []) {
+				$userContext = array_merge($userContext, $externalUserContextData);
```

Comment:
$userContext wird in Zeile 101 mit [] initialisiert, von daher ist es immer gesetzt